fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -231,7 +231,7 @@ dropFrontendClause
 // ------------------------------------------- DML Statement -----------------------------------------------------------
 
 insertStatement
-    : explainDesc? INSERT INTO qualifiedName partitionNames?
+    : explainDesc? INSERT (INTO | OVERWRITE) qualifiedName partitionNames?
```

Comment:
If it is compatible with hive syntax, this should be insert (into | overwrite). However, there is another problem here. I think overwrite should be a reserve keyword, otherwise it cannot be distinguished from the following qualifiedName.

Reply:
as [HangyuanLiu](https://github.com/HangyuanLiu) say, hive/spark use insert (INTO | OVERWRITE)，https://spark.apache.org/docs/3.0.0-preview/sql-ref-syntax-dml-insert-overwrite-table.html

Reply:
@HangyuanLiu ok, I will fix it

Reply:
OK I think it is OK to compatible with Hive and spark.